lab2-alex/syntax.in

```diff
@@ -0,0 +1,19 @@
+declaration = type identifier "=" expression "\n"
+type = ("bool" | "int" | "string")["[]"]
+
+expression = "(" expression operator expression ")" | expression operator expression | expression operator term | "(" expression operator term ")" | term | "input()"
+term = const | identifier
+operator = "+" | "-" | "*" | "/" | "==" | "!=" | "<" | "<=" | ">" | ">=" | "!" | "&" | "|"

```

Comment:
What is "const" here?

Reply:
Added `const = intConst | boolConst | stringConst`